internal/admission/validation/gateway/httproute.go

```diff
@@ -7,8 +7,8 @@ import (
 
 	"github.com/kong/go-kong/kong"
 
-	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/parser"
-	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/parser/translators"
+	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/translator"
+	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/translator/translators"
```

Comment:
`translator/translators` looks a bit confusing. Would there be another renaming to give it a better name? (likely to happen in another PR)

Reply:
good point, I'll address it in separate PR

Reply:
That could be `subtranslators` maybe?